### ISSUE TITLE
api_key arg for openai.Audio.transcribe doesn't work

### ISSUE THREAD

**Issue by dar8900** [openai/openai-python#407]
### Describe the bug

```python
    def create_transcription(self, audio_file : str):
            if not os.path.exists(audio_file):
                return None
            file = open(audio_file, 'rb')
            resp = openai.Audio.transcribe(
                api_key=self._API_KEY,
                file=file,
                model='whisper-1',
                response_format='text',
                language='it'
            )
            return resp
```
Using the **api_key** parameter of `openai.Audio.transcribe` raises a  `openai.error.AuthenticationError`.
I know for sure that the api key provided in **self._API_KEY** is correct  because I have already used it in another function.

Looking in the `audio.py` module, the function:
```python
    def transcribe(
        cls,
        model,
        file,
        api_key=None,
        api_base=None,
        api_type=None,
        api_version=None,
        organization=None,
        **params,
    ):
        requestor, files, data = cls._prepare_request(file, file.name, model, **params)
        url = cls._get_url("transcriptions")
        response, _, api_key = requestor.request("post", url, files=files, params=data)
        return util.convert_to_openai_object(
            response, api_key, api_version, organization
        )
```
does not use the api_key that I am passing it and that it inserts a default key, which is therefore not good.

### To Reproduce

Try to use the function `openai.Audio.transcribe()` with the `api_key` parameter filled with a valid key

### OS

linux

### Python version

Python 3.10.6

### Library version

openai-python-0.27.2

**Comment by KaranAgrawal1509** [comment]
Yes, I faced this too.

Coul you try putting OpenAI key in os.environ and try reading

openai.api_key = os.environ.get("OPENAI_API_KEY")

**Comment by dar8900** [comment]
> Yes, I faced this too.
> 
> Coul you try putting OpenAI key in os.environ and try reading
> 
> openai.api_key = os.environ.get("OPENAI_API_KEY")

yes, this method is what i am currently using and it works fine, thanks

**Comment by LokeshwaranVenugopalBalamurugan** [comment]
> Yes, I faced this too.
> 
> Coul you try putting OpenAI key in os.environ and try reading
> 
> openai.api_key = os.environ.get("OPENAI_API_KEY")

WORKS. THANK YOU!

**Comment by kulapard** [comment]
Fixed in [v0.27.3](https://github.com/openai/openai-python/releases/tag/v0.27.3)

**Comment by dar8900** [comment]
> Fixed in [v0.27.3](https://github.com/openai/openai-python/releases/tag/v0.27.3)

You're right, i've done an update to the latest version and works! Thanks!
